spec/rspec/expectations/failure_aggregator_spec.rb

```diff
@@ -400,6 +415,19 @@ def expect_error_included_in_aggregated_failure(error)
       def fail_including
         fail { |e| expect(e.message).to include(yield) }
       end
+
+      # Each Ruby version return a different exception complement.
+      # This method gets the current version and return the
+      # right complement.
+      def exception_complement(block_levels)
+        engine = RUBY_ENGINE || ''
+        version = RUBY_VERSION.gsub('ruby-', '') || '0'
+        if engine == 'ruby'
```

Comment:
I suggest using `RSpec::Support::Ruby.mri?`.

Reply:
thank you @pirj, it worked :)